### DISCUSSION TITLE
Private email with number or without number?

### DISCUSSION THREAD

**Discussion by kkumlien** [community/community#28957] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi!
On the [email settings](https://github.com/settings/emails), I see my privacy-enabled address myaccount@users.noreply.github.com, but when I hover on the info icon beside the text "Not visible in emails" under my actual email addresses, I see 1234567+myaccount@users.noreply.github.com – I replaced actual values with sample ones.

Which private email should I use for my commits, the former (without number) or the latter (with number)?

Thanks!

**Comment by airtower-luna** [comment, marked as answer]
The difference is described in [Setting your commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) docs:

> If you use your noreply email address for GitHub to make commits and then [change your username](https://docs.github.com/en/articles/changing-your-github-username), those commits will not be associated with your account on GitHub.com. This does not apply if you're using the ID-based noreply address from GitHub.

It also mentions:

> You can get an ID-based noreply email address for GitHub by selecting (or deselecting and reselecting) **Keep my email address private** in your email settings.
